lib/danger/request_sources/github.rb

```diff
@@ -34,16 +30,23 @@ def host
         @host = @environment["DANGER_GITHUB_HOST"] || "github.com"
       end
 
+      # `DANGER_GITHUB_API_HOST` is the old name kept for legacy reasons and
+      # backwards compatibility. `DANGER_GITHUB_API_BASE_URL` is the new
+      # correctly named variable.
       def api_url
-        # `DANGER_GITHUB_API_HOST` is the old name kept for legacy reasons and
-        # backwards compatibility. `DANGER_GITHUB_API_BASE_URL` is the new
-        # correctly named variable.
-        @environment["DANGER_GITHUB_API_HOST"] || @environment["DANGER_GITHUB_API_BASE_URL"]
+        @environment.fetch("DANGER_GITHUB_API_HOST") do
+          @environment.fetch("DANGER_GITHUB_API_BASE_URL") do
+            "https://api.github.com/".freeze
```

Comment:
interesting, so if the lookup fails then it uses the return value from the closure? Smart